### PR TITLE
ipscan: set alias `angryipscanner` for package

### PR DESCRIPTION
**Pull request by totoroot**
## Description of changes

I erroneously set the alias `angryipscanner` for `ipscan` in `pkgs/top-level/aliases.nix` in PR #252363. This change correctly sets alias `angryipscanner` for the package.

@wegank, please review.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -3152,6 +3152,8 @@ with pkgs;
 
   angle-grinder = callPackage ../tools/text/angle-grinder { };
 
+  angryipscanner = callPackage ../tools/security/ipscan { };
```
Comment (wegank) [inline pkgs/top-level/all-packages.nix:3155]: Just put `angryipscanner = ipscan;` under `ipscan = callPackage ...`.

Comment (totoroot) [reply pkgs/top-level/all-packages.nix]: Duh! Thanks, done!

**Review by wegank — APPROVED** [review]

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -9437,6 +9437,7 @@ with pkgs;
   ipinfo = callPackage ../tools/networking/ipinfo { };
 
   ipscan = callPackage ../tools/security/ipscan { };
+  angryipscanner = ipscan;
```
[the remark this answers is not in the document]
Comment (totoroot) [reply pkgs/top-level/all-packages.nix]: @SuperSandro2000 Done!

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -9437,6 +9437,8 @@ with pkgs;
   ipinfo = callPackage ../tools/networking/ipinfo { };
 
   ipscan = callPackage ../tools/security/ipscan { };
+  # `ipscan` is commonly known under the name `angryipscanner`
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:9440]:
```suggestion
  # ipscan is commonly known under the name angryipscanner
```
I wouldn't set those in comments

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**